jsonnet/utils/sanitize-rules.libsonnet

```diff
@@ -176,20 +208,19 @@ local patchOrExcludeRuleGroup(group, groupSet, operation) =
   // 'alert' or 'record' identifier. The function will apply the patch to every
   // alerting/recording rule in the group whose name starts by the identifier.
   patchRules(o): {
-    local patch(o) = o {
-      [if (o.kind == 'PrometheusRule') then 'spec']+: {
-        groups: std.map(
-          function(group)
-            group {
-              rules: std.flattenArrays(
-                patchOrExcludeRuleGroup(group, patchedRules, 'patch')
-              ),
-            },
-          super.groups,
-        ),
-      },
-    },
-    [k]: patch(o[k])
+    [k]: $.patchRule(o[k])
+    for k in std.objectFields(o)
+  },
+
+  // removeRunbookUrl removes runbook_url from alert annotations, but skips alerts that are in includeRunbooks list
+  removeRunbookUrl(o): {
+    [k]: $.removeRunbookAnnotation(o[k])
+    for k in std.objectFields(o)
+  },
+
+  // shorthand for rule patching, rule excluding, and runbook_url removal
+  sanitizeAlertRules(o): {
+    [k]: $.removeRunbookAnnotation($.patchRule($.excludeRule(o[k])))
```

Comment:
This is a slight performance optimization as it will iterate only once over all objects instead of doing this 3 times.